src/assignments/tech-jobs-mvc.rst

```diff
@@ -307,105 +276,56 @@ the bottom of the window. This list is populated by any code comments that
-job information.
-
-Display List of Jobs
-^^^^^^^^^^^^^^^^^^^^^
-
-This page needs to show full job listings---ID, employer, location, etc.
-
-#. In ``Jobs.cshtml``, create a loop to display each job passed in from the
-   controller. The job data should be presented in *tables*, with each job in its own table. Be sure to check in ``ListController`` to find the variable names available to the template.
-#. Adding the CSS class ``"job-listing"`` to a table provides some nice
-   styling, courtesy of Carly’s work!
-#. Construct a table for each job, with
-   one job field per row.
-#. Be sure to test your code by running the program and clicking links from
-   different categories on the ``/list`` page.
-
-   a. Clicking the *Web - Back End* link yields 6 jobs.
-   b. *iOS* yields 4 jobs.
-   c. *New York* yields 1 job.
-   d. Etc.
-
-Add ``View All`` Link
-^^^^^^^^^^^^^^^^^^^^^
+.. admonition:: Note
 
-Open the ``Index.cshtml`` template. This file builds the table to display all of
-the links for the different ``Job`` fields. However, the link for the ``All``
-column is missing.
+   You may not see a ``TODO #4``. This is because TODO comments in views do not always show up in the Task List.
+   If it is not there, check out the ``Search/Index.cshtml`` view to locate it!
 
-.. figure:: figures/listTableOptions.png
-   :alt: Image of the ``/list`` table.
+2) Complete ``ListController``
+------------------------------
 
-You can fix this several different ways, but two options are presented below.
-Note that you only need to implement ONE option, not both.
+Complete the ``Jobs()`` action method in ``ListController``. Right now, it returns a view, but we need to send some details about jobs to that view.
 
-#. Modify ``tableChoices`` in ``ListController`` to include another key/value
-   pair. Check the method that renders the template to help identify the name
-   to use for the key.
-#. Modify ``list.html`` to fill in the empty table cell with the necessary
-   link. Check ``ListController`` to help identify the data to pass in for the
-   query parameters.
+#. The view relies on ``ViewBag.jobs``, so to start create a list in the action method called ``jobs``.
+#. If the user selects "View All", you should use ``JobData.FindAll()`` to populate ``jobs`` with all the jobs and update ``ViewBag.title``. If the user selects something specific, you should use ``JobData.FindJobsByColumnAndValue()`` to populate ``jobs`` with jobs that only match that criteria and update ``ViewBag.title`` to include the criteria the user chose.
+#. Make sure to set ``ViewBag.jobs`` equal to ``jobs`` and run the program to see how it is working now!
 
-Be sure to test your code by clicking your new *View All* link in the table.
-There are 98 jobs in the data file.
+If everything looks good to you, run the tests in ``TestTaskTwo`` in ``AutogradingTests`` to make sure you are on the right track before proceeding to task three.
```

Comment:
Note: On my Mac, I can leave Task 3 tests alone and it will run task 2 tests without any errors.  If i leave task 4 tests uncommented, then it throws errors (See trello card for screen shot).  Once i comment out task 4 tests, it lets me work on the assignment one task at a time.

Reply:
That is so weird! I didn't get that at all. I will work on a fix so it will behave more consistently for students.